File: Fortran/abc197_a/main.f90
program main
   implicit none
   character(len=100) :: input, rotated
   character(len=1) :: head
   character(len=99) :: tail
   integer :: len_input

   read(*, '(A)') input
   len_input = len_trim(input)

   head = input(1:1)
   tail = input(2:len_input)

   rotated = trim(tail) // head
   print '(A)', trim(rotated)
end program main

File: Lua/abc197_a/main.lua
local function read_a_input()
  local input = io.read("*l")
  return input
end

local function solve(input)
  local head = string.sub(input, 0, 1)
  local tail = string.sub(input, 2, 3)
  return tail .. head
end

local function main()
  local input = read_a_input()
  local result = solve(input)
  print(result)
end

main()
